lib/util.js

```diff
@@ -191,59 +192,47 @@ function getLink(obj, linkName, altName) {
   } else {
     return link;
   }
-}
+};
+
+util.getNativeConsole = function() {
+  return window.console;
+};
 
-function warn(text) {
+util.getConsole = function() {
+  var nativeConsole = util.getNativeConsole();
```

Comment:
Double checking - this doesn't throw an error does it? I'm guessing it does not, but we did have issues in the plugin (and I'm mostly guessing the issues in the plugin were related to a long time ago issue where we didn't have access to the window object)